packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -167,7 +168,7 @@ export class ListItem
    *
    * @private
```

Comment:
Sometimes we use `@private` and sometimes `@internal`, is there a preferred one or two distinct use cases? @jcfranco

Reply:
See the Lumina docs here for the differences between `@internal` and `@private`: https://qawebgis.esri.com/arcgis-components/lumina/transition-from-stencil#internal-members